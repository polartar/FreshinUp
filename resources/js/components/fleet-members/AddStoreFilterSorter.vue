<template>
  <div class="px-4 pt-4">
    <v-text-field
      v-model="name"
      label="Search by Fleet Member name"
      single-line
      outline
    />
    <div>
      <div class="pb-4">
        <button
          class="font-weight-bold grey--text"
          @click="toggleShowFilter"
        >
          <span v-if="showFilters">
            <v-flex>
              <v-icon small>fa-caret-down</v-icon>
              Hide Filters
            </v-flex>
          </span>
          <span v-else>
            <v-flex>
              <v-icon small>fa-caret-right</v-icon>
              Show Filters
            </v-flex>
          </span>
        </button>
      </div>
      <div
        v-show="showFilters"
      >
        <v-layout
          row
          wrap
          align-center
          justify-center
        >
          <v-flex
            sm3
            pr-2
          >
            <v-layout
              row
              justify-space-between
              mb-2
            >
              <filter-label
                class="grey--text"
              >
                State
              </filter-label>
              <clear-button
                v-if="state_of_incorporation"
                color="grey"
                @clear="state_of_incorporation = null"
              />
            </v-layout>
            <v-text-field
              v-model="state_of_incorporation"
              label="State of incorporation"
              outline
              single-line
            />
          </v-flex>
          <v-flex
            sm3
            pr-2
          >
            <v-layout
              row
              justify-space-between
              mb-2
            >
              <filter-label
                class="grey--text"
              >
                Type
              </filter-label>
              <clear-button
                v-if="type_id"
                color="grey"
                @clear="type_id = null"
              />
            </v-layout>
            <v-select
              v-model="type_id"
              :items="types"
              item-value="id"
              item-text="name"
              label="Type"
              outline
              single-line
            />
          </v-flex>
          <v-flex
            sm3
            pr-2
          >
            <v-layout
              row
              justify-space-between
              mb-2
            >
              <filter-label class="grey--text">
                Tags
              </filter-label>
              <clear-button
                v-if="tags && tags.length > 0"
                color="grey"
                @clear="tags = []; $refs.tag.resetTerm()"
              />
            </v-layout>
            <f-multi-simple
              ref="tag"
              v-model="tags"
              url="foodfleet/store-tags"
              term-param="filter[name]"
              results-id-key="uuid"
              placeholder="Search Tag"
              background-color="white"
              class="mt-0 pt-0 mb-4"
              height="48"
              not-clearable
              solo
              ouline
              flat
            />
          </v-flex>
          <v-flex
            sm3
          >
            <v-btn
              large
              depressed
              @click="clearFilters"
            >
              Clear all filters
            </v-btn>
          </v-flex>
        </v-layout>
      </div>
    </div>
  </div>
</template>

<script>
import ClearButton from '~/components/ClearButton'
import FilterLabel from '~/components/FilterLabel'
import FMultiSimple from 'fresh-bus/components/ui/FMultiSimple'
import MapValueKeysToData from '../../mixins/MapValueKeysToData'
export const DEFAULT_FILTERS = {
  name: '',
  state_of_incorporation: '',
  type_id: '',
  tags: []
}
export default {
  components: {
    ClearButton,
    FilterLabel,
    FMultiSimple
  },
  mixins: [MapValueKeysToData],
  props: {
    types: { type: Array, default: () => [] },
    // overriding value props from MapValueKeysToData mixin
    // to provide default value
    value: { type: Object, default: () => DEFAULT_FILTERS }
  },
  data () {
    return {
      showFilters: false,
      ...DEFAULT_FILTERS
    }
  },
  watch: {
    payload () {
      this.run()
    }
  },
  methods: {
    clearFilters () {
      this.name = ''
      this.state_of_incorporation = ''
      this.type_id = ''
      this.tags = []
      this.$refs.tag.resetTerm()
    },
    toggleShowFilter () {
      this.showFilters = !this.showFilters
    },
    run () {
      this.$emit('input', {
        name: this.payload.name,
        state_of_incorporation: this.payload.state_of_incorporation,
        type_id: this.payload.type_id,
        tag: this.tags ? this.tags.map(item => item.uuid) : null
      })
    }
  }
}
</script>
