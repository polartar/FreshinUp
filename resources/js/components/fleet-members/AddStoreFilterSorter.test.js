import { mount, shallowMount } from '@vue/test-utils'
import { FIXTURE_STORE_TAGS } from '../../../../tests/Javascript/__data__/storeTags'
import Component from './AddStoreFilterSorter'
import * as Stories from './AddStoreFilterSorter.stories'

describe('components/fleet-members/AddStoreFilterSorter', () => {

  describe('Snapshots', () => {
    test('Default', async () => {
      const wrapper = mount(Stories.Default())
      await wrapper.vm.$nextTick()
      expect(wrapper.element).toMatchSnapshot()
    })
    test('Populated', async () => {
      const wrapper = mount(Stories.Populated())
      await wrapper.vm.$nextTick()
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  describe('Methods', () => {

    test('showFilters()', () => {
      const wrapper = shallowMount(Component)

      wrapper.setData({
        showFilters: false
      })

      expect(wrapper.vm.showFilters).toBeFalsy()

      wrapper.vm.toggleShowFilter()
      expect(wrapper.vm.showFilters).toBeTruthy()

      wrapper.vm.toggleShowFilter()
      expect(wrapper.vm.showFilters).toBeFalsy()
    })

    test('clearAllFilters()', async () => {
      const wrapper = shallowMount(Component)

      wrapper.setProps({
        value: {
          state_of_incorporation: 'aaa',
          type_id: 2,
          tags: FIXTURE_STORE_TAGS
        }
      })
      await wrapper.vm.$nextTick()
      const mock = jest.fn()
      wrapper.vm.$refs = {
        tag: {
          resetTerm: mock
        }
      }
      wrapper.vm.clearFilters()
      expect(mock).toHaveBeenCalled()
      expect(wrapper.vm.name).toBe('')
      expect(wrapper.vm.state_of_incorporation).toBe('')
      expect(wrapper.vm.type_id).toBe('')
      expect(wrapper.vm.tags).toEqual([])
    })
  })
})
