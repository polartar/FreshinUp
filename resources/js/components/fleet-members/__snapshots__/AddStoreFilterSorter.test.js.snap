// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/fleet-members/AddStoreFilterSorter Snapshots Default 1`] = `
<div
  class="px-4 pt-4"
>
  <div
    class="v-input v-text-field v-text-field--single-line v-text-field--enclosed v-text-field--outline theme--light"
  >
    <div
      class="v-input__control"
    >
      <div
        class="v-input__slot"
      >
        <div
          class="v-text-field__slot"
        >
          <label
            aria-hidden="true"
            class="v-label theme--light"
            style="left: 0px; position: absolute;"
          >
            Search by Fleet Member name
          </label>
          <input
            aria-label="Search by Fleet Member name"
            type="text"
          />
        </div>
      </div>
      <div
        class="v-text-field__details"
      >
        <div
          class="v-messages theme--light"
        >
          <div
            class="v-messages__wrapper"
          />
        </div>
      </div>
    </div>
  </div>
   
  <div>
    <div
      class="pb-4"
    >
      <button
        class="font-weight-bold grey--text"
      >
        <span>
          <div
            class="flex"
          >
            <i
              aria-hidden="true"
              class="v-icon fa fa-caret-right theme--light"
              style="font-size: 16px;"
            />
            
            Show Filters
          
          </div>
        </span>
      </button>
    </div>
     
    <div
      style="display: none;"
    >
      <div
        class="layout row wrap align-center justify-center"
      >
        <div
          class="flex sm3 pr-2"
        >
          <div
            class="layout row justify-space-between mb-2"
          >
            <div
              class="grey--text white--text"
            >
              <strong
                class="no-wrap"
              >
                
              State
            
              </strong>
            </div>
             
            <!---->
          </div>
           
          <div
            class="v-input v-text-field v-text-field--single-line v-text-field--enclosed v-text-field--outline theme--light"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot"
              >
                <div
                  class="v-text-field__slot"
                >
                  <label
                    aria-hidden="true"
                    class="v-label theme--light"
                    style="left: 0px; position: absolute;"
                  >
                    State of incorporation
                  </label>
                  <input
                    aria-label="State of incorporation"
                    type="text"
                  />
                </div>
              </div>
              <div
                class="v-text-field__details"
              >
                <div
                  class="v-messages theme--light"
                >
                  <div
                    class="v-messages__wrapper"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="flex sm3 pr-2"
        >
          <div
            class="layout row justify-space-between mb-2"
          >
            <div
              class="grey--text white--text"
            >
              <strong
                class="no-wrap"
              >
                
              Type
            
              </strong>
            </div>
             
            <!---->
          </div>
           
          <div
            class="v-input v-text-field v-text-field--single-line v-text-field--enclosed v-text-field--outline v-select theme--light"
            role="combobox"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot"
              >
                <div
                  class="v-select__slot"
                >
                  <label
                    aria-hidden="true"
                    class="v-label theme--light"
                    style="left: 0px; position: absolute;"
                  >
                    Type
                  </label>
                  <div
                    class="v-select__selections"
                  >
                    <input
                      aria-label="Type"
                      aria-readonly="false"
                      autocomplete="on"
                      readonly="readonly"
                      type="text"
                    />
                  </div>
                  <div
                    class="v-input__append-inner"
                  >
                    <div
                      class="v-input__icon v-input__icon--append"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon material-icons theme--light"
                      >
                        arrow_drop_down
                      </i>
                    </div>
                  </div>
                </div>
                <div
                  class="v-menu"
                >
                  <div
                    class="v-menu__content theme--light "
                    style="max-height: 300px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                  >
                    <div
                      class="v-select-list v-card theme--light"
                    >
                      <div
                        class="v-list theme--light"
                        role="list"
                      >
                        <div
                          role="listitem"
                        >
                          <div
                            class="v-list__tile theme--light"
                          >
                            <div
                              class="v-list__tile__content"
                            >
                              <div
                                class="v-list__tile__title"
                              >
                                No data available
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="v-text-field__details"
              >
                <div
                  class="v-messages theme--light"
                >
                  <div
                    class="v-messages__wrapper"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="flex sm3 pr-2"
        >
          <div
            class="layout row justify-space-between mb-2"
          >
            <div
              class="grey--text white--text"
            >
              <strong
                class="no-wrap"
              >
                
              Tags
            
              </strong>
            </div>
             
            <!---->
          </div>
           
          <div
            class="v-input v-text-field v-text-field--single-line v-text-field--solo v-text-field--solo-flat v-text-field--enclosed v-text-field--placeholder v-input--hide-details v-input--is-readonly theme--light"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot white"
                style="height: 48px;"
              >
                <div
                  class="v-text-field__slot"
                >
                  <input
                    not-clearable=""
                    ouline=""
                    placeholder="Search Tag"
                    readonly="readonly"
                    results-id-key="uuid"
                    term-param="filter[name]"
                    type="text"
                    url="foodfleet/store-tags"
                  />
                </div>
                <div
                  class="v-input__append-inner"
                >
                  <div
                    class="v-input__icon v-input__icon--append"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon fas fa-caret-down theme--light"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            background-color="white"
            class="v-menu mt-0 pt-0 mb-4"
            flat=""
            height="48"
            not-clearable=""
            ouline=""
            placeholder="Search Tag"
            results-id-key="uuid"
            solo=""
            term-param="filter[name]"
            url="foodfleet/store-tags"
          >
            <div
              class="v-menu__content theme--light "
              style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
            >
               
              <div
                autofocus="autofocus"
                background-color="white"
                class="v-input v-text-field v-text-field--single-line v-text-field--solo v-text-field--solo-flat v-text-field--enclosed v-text-field--placeholder v-select v-autocomplete v-input--hide-details v-input--is-focused theme--light primary--text"
                flat=""
                height="48"
                menu-props="[object Object]"
                ouline=""
                solo=""
              >
                <div
                  class="v-input__control"
                >
                  <div
                    class="v-input__slot white"
                    style="height: 48px;"
                  >
                    <div
                      class="v-select__slot"
                    >
                      <div
                        class="v-select__selections"
                      >
                        <input
                          aria-label=""
                          autocomplete="off"
                          autofocus="autofocus"
                          ouline=""
                          placeholder="Search Tag"
                          role="combobox"
                          type="text"
                        />
                      </div>
                      <div
                        class="v-input__append-inner"
                      >
                        <div
                          class="v-input__icon v-input__icon--append"
                        >
                          <i
                            aria-hidden="true"
                            class="v-icon material-icons theme--light primary--text"
                          >
                            arrow_drop_down
                          </i>
                        </div>
                      </div>
                    </div>
                    <div
                      class="v-menu"
                    >
                      <div
                        class="v-menu__content theme--light v-autocomplete__content"
                        style="max-height: 300px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                      >
                        <div
                          class="v-select-list v-card theme--light"
                        >
                          <div
                            class="v-list theme--light"
                            role="list"
                          >
                            <div
                              role="listitem"
                              style="display: none;"
                            >
                              <div
                                class="v-list__tile theme--light"
                              >
                                <span
                                  class="search-section-title"
                                >
                                  SELECTION
                                </span>
                              </div>
                            </div>
                              
                            <div
                              role="listitem"
                              style="display: none;"
                            >
                              <div
                                class="v-list__tile theme--light"
                              >
                                <span
                                  class="search-section-title"
                                >
                                  RESULTS
                                </span>
                              </div>
                            </div>
                            <div
                              role="listitem"
                            >
                              <div
                                class="v-list__tile theme--light"
                              >
                                <div
                                  class="v-list__tile__content"
                                >
                                  <div
                                    class="v-list__tile__title"
                                  >
                                    No data available
                                  </div>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="flex sm3"
        >
          <button
            class="v-btn v-btn--large v-btn--depressed theme--light"
            type="button"
          >
            <div
              class="v-btn__content"
            >
              
            Clear all filters
          
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`components/fleet-members/AddStoreFilterSorter Snapshots Populated 1`] = `
<div
  class="px-4 pt-4"
>
  <div
    class="v-input v-text-field v-text-field--single-line v-text-field--enclosed v-text-field--outline v-input--is-label-active v-input--is-dirty theme--light"
  >
    <div
      class="v-input__control"
    >
      <div
        class="v-input__slot"
      >
        <div
          class="v-text-field__slot"
        >
          <input
            aria-label="Search by Fleet Member name"
            type="text"
          />
        </div>
      </div>
      <div
        class="v-text-field__details"
      >
        <div
          class="v-messages theme--light"
        >
          <div
            class="v-messages__wrapper"
          />
        </div>
      </div>
    </div>
  </div>
   
  <div>
    <div
      class="pb-4"
    >
      <button
        class="font-weight-bold grey--text"
      >
        <span>
          <div
            class="flex"
          >
            <i
              aria-hidden="true"
              class="v-icon fa fa-caret-right theme--light"
              style="font-size: 16px;"
            />
            
            Show Filters
          
          </div>
        </span>
      </button>
    </div>
     
    <div
      style="display: none;"
    >
      <div
        class="layout row wrap align-center justify-center"
      >
        <div
          class="flex sm3 pr-2"
        >
          <div
            class="layout row justify-space-between mb-2"
          >
            <div
              class="grey--text white--text"
            >
              <strong
                class="no-wrap"
              >
                
              State
            
              </strong>
            </div>
             
            <button
              class="mr-0 v-btn v-btn--flat v-btn--icon theme--light grey--text"
              color="grey"
              type="button"
            >
              <div
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon clear-button-icon fa fa fa fa-times theme--light"
                />
              </div>
            </button>
          </div>
           
          <div
            class="v-input v-text-field v-text-field--single-line v-text-field--enclosed v-text-field--outline v-input--is-label-active v-input--is-dirty theme--light"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot"
              >
                <div
                  class="v-text-field__slot"
                >
                  <input
                    aria-label="State of incorporation"
                    type="text"
                  />
                </div>
              </div>
              <div
                class="v-text-field__details"
              >
                <div
                  class="v-messages theme--light"
                >
                  <div
                    class="v-messages__wrapper"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="flex sm3 pr-2"
        >
          <div
            class="layout row justify-space-between mb-2"
          >
            <div
              class="grey--text white--text"
            >
              <strong
                class="no-wrap"
              >
                
              Type
            
              </strong>
            </div>
             
            <button
              class="mr-0 v-btn v-btn--flat v-btn--icon theme--light grey--text"
              color="grey"
              type="button"
            >
              <div
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon clear-button-icon fa fa fa fa-times theme--light"
                />
              </div>
            </button>
          </div>
           
          <div
            class="v-input v-text-field v-text-field--single-line v-text-field--enclosed v-text-field--outline v-select v-input--is-label-active v-input--is-dirty theme--light"
            role="combobox"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot"
              >
                <div
                  class="v-select__slot"
                >
                  <div
                    class="v-select__selections"
                  >
                    <div
                      class="v-select__selection v-select__selection--comma"
                    >
                      Mobile
                    </div>
                    <input
                      aria-label="Type"
                      aria-readonly="false"
                      autocomplete="on"
                      readonly="readonly"
                      type="text"
                    />
                  </div>
                  <div
                    class="v-input__append-inner"
                  >
                    <div
                      class="v-input__icon v-input__icon--append"
                    >
                      <i
                        aria-hidden="true"
                        class="v-icon material-icons theme--light"
                      >
                        arrow_drop_down
                      </i>
                    </div>
                  </div>
                </div>
                <div
                  class="v-menu"
                >
                  <div
                    class="v-menu__content theme--light "
                    style="max-height: 300px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                  >
                    <div
                      class="v-select-list v-card theme--light"
                    >
                      <div
                        class="v-list theme--light"
                        role="list"
                      >
                        <div
                          class="primary--text"
                          role="listitem"
                        >
                          <a
                            class="v-list__tile v-list__tile--link v-list__tile--active theme--light primary--text"
                          >
                            <div
                              class="v-list__tile__content"
                            >
                              <div
                                class="v-list__tile__title"
                              >
                                Mobile
                              </div>
                            </div>
                          </a>
                        </div>
                        <div
                          role="listitem"
                        >
                          <a
                            class="v-list__tile v-list__tile--link theme--light"
                          >
                            <div
                              class="v-list__tile__content"
                            >
                              <div
                                class="v-list__tile__title"
                              >
                                Stationery
                              </div>
                            </div>
                          </a>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="v-text-field__details"
              >
                <div
                  class="v-messages theme--light"
                >
                  <div
                    class="v-messages__wrapper"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="flex sm3 pr-2"
        >
          <div
            class="layout row justify-space-between mb-2"
          >
            <div
              class="grey--text white--text"
            >
              <strong
                class="no-wrap"
              >
                
              Tags
            
              </strong>
            </div>
             
            <button
              class="mr-0 v-btn v-btn--flat v-btn--icon theme--light grey--text"
              color="grey"
              type="button"
            >
              <div
                class="v-btn__content"
              >
                <i
                  aria-hidden="true"
                  class="v-icon clear-button-icon fa fa fa fa-times theme--light"
                />
              </div>
            </button>
          </div>
           
          <div
            class="v-input v-text-field v-text-field--single-line v-text-field--solo v-text-field--solo-flat v-text-field--enclosed v-text-field--placeholder v-input--hide-details v-input--is-label-active v-input--is-dirty v-input--is-readonly theme--light"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot white"
                style="height: 48px;"
              >
                <div
                  class="v-text-field__slot"
                >
                  <input
                    not-clearable=""
                    ouline=""
                    placeholder="Search Tag"
                    readonly="readonly"
                    results-id-key="uuid"
                    term-param="filter[name]"
                    type="text"
                    url="foodfleet/store-tags"
                  />
                </div>
                <div
                  class="v-input__append-inner"
                >
                  <div
                    class="v-input__icon v-input__icon--append"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon fas fa-caret-down theme--light"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            background-color="white"
            class="v-menu mt-0 pt-0 mb-4"
            flat=""
            height="48"
            not-clearable=""
            ouline=""
            placeholder="Search Tag"
            results-id-key="uuid"
            solo=""
            term-param="filter[name]"
            url="foodfleet/store-tags"
          >
            <div
              class="v-menu__content theme--light "
              style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
            >
               
              <div
                autofocus="autofocus"
                background-color="white"
                class="v-input v-text-field v-text-field--single-line v-text-field--solo v-text-field--solo-flat v-text-field--enclosed v-text-field--placeholder v-select v-autocomplete v-input--hide-details v-input--is-focused theme--light primary--text"
                flat=""
                height="48"
                menu-props="[object Object]"
                ouline=""
                solo=""
              >
                <div
                  class="v-input__control"
                >
                  <div
                    class="v-input__slot white"
                    style="height: 48px;"
                  >
                    <div
                      class="v-select__slot"
                    >
                      <div
                        class="v-select__selections"
                      >
                        <input
                          aria-label=""
                          autocomplete="off"
                          autofocus="autofocus"
                          ouline=""
                          placeholder="Search Tag"
                          role="combobox"
                          type="text"
                        />
                      </div>
                      <div
                        class="v-input__append-inner"
                      >
                        <div
                          class="v-input__icon v-input__icon--append"
                        >
                          <i
                            aria-hidden="true"
                            class="v-icon material-icons theme--light primary--text"
                          >
                            arrow_drop_down
                          </i>
                        </div>
                      </div>
                    </div>
                    <div
                      class="v-menu"
                    >
                      <div
                        class="v-menu__content theme--light v-autocomplete__content"
                        style="max-height: 300px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
                      >
                        <div
                          class="v-select-list v-card theme--light"
                        >
                          <div
                            class="v-list theme--light"
                            role="list"
                          >
                            <div
                              role="listitem"
                              style="display: none;"
                            >
                              <div
                                class="v-list__tile theme--light"
                              >
                                <span
                                  class="search-section-title"
                                >
                                  SELECTION
                                </span>
                              </div>
                            </div>
                              
                            <div
                              role="listitem"
                              style="display: none;"
                            >
                              <div
                                class="v-list__tile theme--light"
                              >
                                <span
                                  class="search-section-title"
                                >
                                  RESULTS
                                </span>
                              </div>
                            </div>
                            <div
                              role="listitem"
                            >
                              <div
                                class="v-list__tile theme--light"
                              >
                                <div
                                  class="v-list__tile__content"
                                >
                                  <div
                                    class="v-list__tile__title"
                                  >
                                    No data available
                                  </div>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="flex sm3"
        >
          <button
            class="v-btn v-btn--large v-btn--depressed theme--light"
            type="button"
          >
            <div
              class="v-btn__content"
            >
              
            Clear all filters
          
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
